google/ggrc-core
# GGRC-4367 Object review - 'Submit for review' link is duplicated

# Issue description

'Submit for review' link is duplicated after filling out custom attributes via edit modal popup

# Steps to test the changes

Preconditions:
Have any global custom attribute for Program, e.g. Rich text

Steps to reproduce:
1. Create a program
2. Open Edit Program popup and fill out global custom attribute > Save
3. Look at the 'Submit for review' link;

_Actual Result_: 'Submit for review' link is duplicated after filling out custom attributes via edit modal popup
_Expected Result_: 'Submit for review' link should be duplicated after filling out custom attributes via edit modal popup

[https://ggrc-dev.appspot.com/programs/3#!info_widget/program/3](
https://ggrc-dev.appspot.com/programs/3#!info_widget/program/3)

# Solution description

Add "review-link" component instead of using "with_mapping_count" helper

# Sanity checklist




# PR Review checklist
